modules/main-rules/pages/transport.adoc

```diff
@@ -1,39 +1,61 @@
+On some unit data sheets you'll see that the unit has one of two special abilities that relate to transportation: {transport} or {hitch}.
+We explain these and other special abilities in more detail elsewhere; but transport is a vital aspect of the Movement phase so we'll discuss it here too.
 
-Units that have the {transport} special ability can carry other units across the battlefield.
+When we speak of these transportation abilities, we use the terms 'embark/embarked' and 'disembark/disembarked' to describe actions and states that relate to the units that you transport.
 
-* To embark or disembark other units (or cargo) it costs the {transport} unit 5cm of its movement.
-* An embarking or disembarking unit must also pay 5cm of its movement.
-* You may embark and disembark units in the same turn, the same phase or even in the same move -- so long as all of the involved units have enough movement.
-* You may perform no other actions with units that are embarked -- so, you cannot shoot with them, for example. 
+Both the {transport} and {hitch} abilities indicate units that can carry other units across the battlefield, but they work in slightly different ways.
+
+== Transportation by units that have the {transport} ability
+
+Units that have the {transport} ability can carry other units across the battlefield.
+They can only transport Infantry units -- except where the rules or army lists say otherwise.
+
+To embark or disembark a unit:
+
+* You must sacrifice 5 cm from the movement of the {transport} unit.
+* You must sacrifice 5 cm from the movement of the unit that you embark or disembark.
+* You may embark and disembark units in the same turn, the same phase or even in the same move -- so long as all of the involved units have enough movement allowance.
+
+You may perform no other actions with units that are embarked -- so, you cannot shoot with them, for example.
 Obviously, once you disembark them then the normal rules apply again.
 
+[NOTE]
+.Placement of disembarked units
+====
+When you disembark units, measure their movement from the base of the {transport} unit.
+If you do not have enough movement to clear the {transport} unit's base then place the disembarked unit in base contact with the {transport} unit.
+====
+
+[NOTE.e40k]
+.Placement of disembarked units
+====
+We added this rule to minimise the effects that result from the assortment of different base sizes that players use.
+====
+
 == Destruction of {transport} units
 
-If a {transport} unit is destroyed then, just before you remove the model as a casualty, roll a D6 for each embarked unit:
+If a {transport} unit is destroyed then, just before you remove the unit as a casualty, roll a D6 for each embarked unit:
 
-* *1–3*: The embarked unit is destroyed along with the {transport} unit.
+* *1–3*: The embarked unit suffers a hit.
 * *4–6*: The embarked unit survives -- place it in contact with the destroyed {transport} unit.
```

Comment:
This is in reference to the two results of the die roll.

At some point (probably not now), this needs to be restructured so that placement of survivors is not linked to a specific result, but applies to anyone who survives. This will stop units with Save or multiple Damage Capacity from needing extra text.

That's partly why I would have liked to swap it from '4+ to live' to '4+ to get hit', as it allows for a much more concise structure like this:

... roll a D6 for each embarked unit. On a roll of 4+, the unit suffers a hit. If the unit survives, place it in contact with the destroyed {transport} unit.

Reply:
I see what you mean. Plus, in general I find it much easier to write such rules in a consistent way if we phrase such that the opponent rolls for this type of 'bad stuff' against you. Then the normal '4+/high numbers are good' expectation is met because it is 'good' from your opponent's point of view, just as it is when they roll to hit with their own shooting. Which you've probably thought of too. After all, it doesn't change the actual mechanic, it's still a 50:50 roll.

So, I agree it'd be nice not to hold up this PR for it, but I'm also supportive of the idea that we could try a rewrite of this bit later to 'invert' the result if it will make other elements more tidy and consistent.

Reply:
Of course, an option that would slot straight into the existing text:

```suggestion
On a roll of 1 to 3, the unit suffers a hit.
If the unit survives, place it in contact with the destroyed {transport} unit.
```